api/tests/devcomp/unit/infrastructure/datasources/learning-content/validation/module-validation_test.js

```diff
@@ -0,0 +1,65 @@
+import { expect } from '../../../../../../test-helper.js';
+import {
+  imageElementSchema,
+  qcmElementSchema,
+  qcuElementSchema,
+  qrocmElementSchema,
+  textElementSchema,
+  videoElementSchema,
+} from './element/index.js';
+import { getImageSample } from '../../../../../../../src/devcomp/infrastructure/datasources/learning-content/samples/elements/image.sample.js';
+import { getQcmSample } from '../../../../../../../src/devcomp/infrastructure/datasources/learning-content/samples/elements/qcm.sample.js';
+import { getQcuSample } from '../../../../../../../src/devcomp/infrastructure/datasources/learning-content/samples/elements/qcu.sample.js';
+import { getQrocmSample } from '../../../../../../../src/devcomp/infrastructure/datasources/learning-content/samples/elements/qrocm.sample.js';
+import { getTextSample } from '../../../../../../../src/devcomp/infrastructure/datasources/learning-content/samples/elements/text.sample.js';
+import { getVideoSample } from '../../../../../../../src/devcomp/infrastructure/datasources/learning-content/samples/elements/video.sample.js';
+
+describe('Unit | Infrastructure | Datasources | Learning Content | Module Datasource | format validation', function () {
+  it('should validate sample image structure', function () {
+    // When
+    const result = imageElementSchema.validate(getImageSample());
+
+    // Then
+    expect(result.error).to.equal(undefined, result.error?.details.map((error) => error.message).join('. '));
```

Comment:
En regardant de plus près avec @mariannebost et @mirawlin, nous voyons pas vraiment l'intérêt de nommer l'erreur : 
Voici les erreurs pour chaque expect : 

```
expect(result.error).to.equal(undefined, result.error?.details.map((error) => error.message).join('. '));
  // AssertionError: "alt" is required: expected ValidationError: "alt" is required { …(2) } to equal undefined
```

```
expect(result.error).to.equal(undefined);
    // AssertionError: expected ValidationError: "alt" is required { …(2) } to equal undefined
```

```
expect(result.error).to.be.undefined;
    // AssertionError: expected ValidationError: "alt" is required { …(2) } to be undefined
```

Comme cela complexifie vachement la lecture des expects : il faut savoir que le deuxième paramètre est un override de l'erreur, et que ` result.error?.details.map((error) => error.message).join('. ')` a pour but d'afficher toutes les erreurs du schéma. Nous proposons plutôt de rester simple avec : 
```suggestion
		expect(result.error).to.be.undefined; 
```

Reply:
Pour info c'est un copié collé de ce qui est fait côté module-datasource_test.

Chaud pour voir ensemble comment faire mieux !